diff --git a/buildtools/third_party/libc++/BUILD.gn b/buildtools/third_party/libc++/BUILD.gn
index 301d2fd..2fdeb20 100644
--- a/buildtools/third_party/libc++/BUILD.gn
+++ b/buildtools/third_party/libc++/BUILD.gn
@@ -51,6 +51,11 @@ target(_libcxx_target_type, "libc++") {
   if (libcxx_is_shared) {
     no_default_deps = true
   }
+
+  if (!is_clang) {
+    libs = [ "atomic" ]
+  }
+
   sources = [
     "trunk/src/algorithm.cpp",
     "trunk/src/any.cpp",
