diff --git a/build/build_config.h b/build/build_config.h
index c37c7429..82a991e 100644
--- a/build/build_config.h
+++ b/build/build_config.h
@@ -155,6 +155,11 @@
 #define ARCH_CPU_PPC64 1
 #define ARCH_CPU_64_BITS 1
 #define ARCH_CPU_BIG_ENDIAN 1
+#elif defined(__riscv) && (__riscv_xlen == 64)
+#define ARCH_CPU_RISCV_FAMILY 1
+#define ARCH_CPU_RISCV64 1
+#define ARCH_CPU_64_BITS 1
+#define ARCH_CPU_LITTLE_ENDIAN 1
 #elif defined(__PPC64__)
 #define ARCH_CPU_PPC64_FAMILY 1
 #define ARCH_CPU_PPC64 1
